docs/BUILD.bazel

```diff
@@ -368,6 +369,9 @@ genrule(
         # Copy templates for code snippets in getting started guide
         mkdir -p build/docs/source/getting-started/code
         tar -zxf $(location //templates:templates-tarball) -C build/docs/source/getting-started/code/
+        cd build/docs/source/getting-started/code/templates-tarball
+        ../../../../../../$(location @patch_dev_env//:patch) -s -p1 < ../../../../../../$(location //templates:messaging.patch)
```

Comment:
Hmm I can probably improve this 😄

Reply:
There is an easy solution: Add `PATCH=$PWD/$(location @patch/dev_env//:patch)` before the `cd` and then use `$PATCH` here.  You can also simplify the `cd foo; x; cd -` pattern to `(cd foo; x)` which runs `x` in a subshell and automatically restores the working directory afterwards.

Reply:
Refactored this section, hopefully nicer now